site/src/sphinx/server-annotated-service.rst

```diff
@@ -1291,3 +1291,56 @@ because they always have an empty content.
 | 304          | Not modified   |
 +--------------+----------------+
 
+Using a composite annotation
+----------------------------
+
+In some cases, you may want to make a composite annotation which includes several types of annotations.
```

Comment:
To avoid specifying a common set of annotations repetitively, you may want to create a composite annotation which is annotated by other annotations.